modules/ROOT/pages/Development/Satisfactory/Audio.adoc

```diff
@@ -5,20 +5,367 @@
+
+Once the AK event has ended, the temporary `AK Component` will be automatically cleaned up.
+
+==== Looping Events
+
+If you would like to play a sound effect that might loop, for example, a humming/buzzing sound for a building, you should use the same AK Component for ease of reference and to avoid the creation/destruction overhead. Add an `AK Component` to your actor blueprint and set the `AK Audio Event` variable on the component.
+
+Make sure to stop the audio when the actor is destroyed,
+otherwise it will continue to play with no way to stop it.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_EndPlay.png[Wwise_UEAkEvent_EndPlay, 100%]
+
+=== Updating Wwise Values From Your Mod
+
+Wwise uses a feature called Real-Time Parameter Control (RTPC) that allows game engines like Unreal or Unity to change values on Wwise assets in real-time.
+This enables special behaviors like a manufacturing building's sound volume to fluctuate based on recipe progress.
+
+After setting up RTPC in Wwise and Unreal, blueprint code can update audio parameter values.
+
+To set these up, in your Wwise project, go to Project Explorer and the `Game Syncs` tab.
+
+Under `Game Parameters -> Default Work Unit` create a new game parameter by right-clicking the `Default Work Unit` and then `New Child -> Game Parameter`
+
+image:Satisfactory/Wwise/Wwise_RTPC_Create.png[Wwise_RTPC_Create, 100%]
+
+Name this whatever you want.
+
+Open the newly created Game Parameter, and you should see the following in the main section:
+
+image:Satisfactory/Wwise/Wwise_RTPC_Range.png[Wwise_RTPC_Range, 100%]
+
+The parameter must have a range defined.
+A useful range is `0.0` to `1.0` since many existing gameand engine systems already use that range as a start-finish, minimum-maximum, or off-on scale.
+
+Now, assign the RTPC Game Parameter to your audio.
+
+Open your Audio in Wwise and go to the `RTPC` tab.
+Click the `>>` in the main section to set what you want to update.
+This example case shows updating the `Voice Volume`.
+
+Then click the `>>` to the right of the first one and select your game parameter.
+
+image:Satisfactory/Wwise/Wwise_RTPC_Configure.png[Wwise_RTPC_Configure, 500]
+
+Once that is done, you should see something similar to the following:
+
+image:Satisfactory/Wwise/Wwise_RTPC_Overview.png[Wwise_RTPC_Overview, 100%]
+
+Now, to create the link between Unreal Engine and Wwise you will need to create the Game Parameter in Unreal Engine:
+
+image:Satisfactory/Wwise/Wwise_UE_RTPC_Create.png[Wwise_UE_RTPC_Create, 600]
+
+Name it the same as you have in Wwise.
+
+You can now update the RTPC Game Parameter value using blueprints:
+
+image:Satisfactory/Wwise/Wwise_UE_RTPC_Usage.png[Wwise_UE_RTPC_Usage, 700]
+
+=== Playback Limits
+
+Wwise limits the amount of audio that can be played at once. 
+Trying to play more than this limit will result in lower-priority sounds like footsteps, music, or animal sounds playing.
```

Comment:
Did this get flipped? I would expect higher-priority sounds to be played in this situation, not the lower priority ones.